pkg/inspector/image-inspector.go

```diff
@@ -636,16 +590,188 @@ func createOutputDir(dirName string, tempName string) (string, error) {
 		err := osMkdir(dirName, 0755)
 		if err != nil {
 			if !os.IsExist(err) {
-				return "", fmt.Errorf("Unable to create destination path: %v\n", err)
+				return "", fmt.Errorf("creating destination path: %v\n", err)
 			}
 		}
 	} else {
 		// forcing to use /var/tmp because often it's not an in-memory tmpfs
 		var err error
 		dirName, err = ioutilTempDir("/var/tmp", tempName)
 		if err != nil {
-			return "", fmt.Errorf("Unable to create temporary path: %v\n", err)
+			return "", fmt.Errorf("creating temporary path: %v\n", err)
 		}
 	}
 	return dirName, nil
 }
+
+// pullImage pulls the image using containers/image library
+// to directly connect to the docker registry
+// and will return its metadata (*types.ImageInspectInfo).
+// Assumes the image to be pulled is with a transport prefix
+// or if not attempts to add the "docker://" prefix to the image.
+func (i *defaultImageInspector) pullImage() (*types.ImageInspectInfo, digest.Digest, error) {
```

Comment:
I suggest changing to `containersPullImage` and change `dockerPullImage` to 'dockerSocketPullImage`

Reply:
:+1:  for containersPullImage but I think I will leave `dockerPullImage` as is.